src/graphql/storage-explorer.ts

```diff
@@ -0,0 +1,40 @@
+import { Field, ObjectType } from 'type-graphql';
+
+@ObjectType()
+export class File {
+  @Field((type) => String, { nullable: false })
```

Comment:
think in those files we can use /* eslint-disable @typescript-eslint/no-unused-vars */ to avoid such kind of warnings

Reply:
FIxed globally with eslintrc rule